shared/src/main/java/io/pravega/shared/segment/StreamSegmentNameUtils.java

```diff
@@ -194,6 +182,18 @@ public static String getSegmentChunkName(String segmentName, long offset) {
         return segmentName + OFFSET_SUFFIX + Long.toString(offset);
     }
 
+    /**
+     * Gets the name of the Segment that is used to store the Container's Segment Metadata. There is one such Segment
+     * per container.
+     *
+     * @param containerId The Id of the Container.
+     * @return The Metadata Segment name.
+     */
+    public static String getMetadataSegmentName(int containerId) {
```

Comment:
I know this method is trivial, but for completeness, can we add a test case to `StreamSegmentNameUtilsTests` to exercize this method?

Reply:
Added